### PR TITLE
REXCHAIN Implement support for rediss (TLS) connections

### PR DESCRIPTION
**Pull request by ebarojas**
Move away from temp va and allow for both TLS and non TLS connections

### CODE REVIEW

**Comment by ebarojas** [comment]
Jason, hay que probar que se generen recetas en rexchain-dev - ie: desde prescrypto-development

**Comment by JasonBerto** [comment]
no tengo permisos para cambiar los labels

File: rexchain/rexchain/settings.py [hunk 1]
```diff
@@ -193,20 +196,32 @@
...
+                    "max_connections": 10 # Note, we should review this.
             }
         }
     }
 }
+
+# Parsing URL in order to single out TSL connections
+parsed_url = urlparse(REDIS_URL)
+
+# Append SSL/TLS configuration if using rediss:// (TLS)
```
Comment (mevangelista-alvarado) [inline rexchain/rexchain/settings.py:219]: 👍

**Review by mevangelista-alvarado — APPROVED** [review]
LGTM

**Pull request merged**